app/experimenter/templates/experiments/detail_ship.html

```diff
@@ -41,33 +41,35 @@ <h5 class="modal-title" id="normandyModalLabel">Normandy Recipe</h5>
         <div class="modal-body">
           <p>Enter the following information into Delivery Console exactly as shown:</p>
           <pre class="bg-light json prettyprint">{{ experiment.normandy_recipe_json }}</pre>
-          <p>When the recipe is saved in Delivery Console, copy the Recipe ID below and hit save.</p>
-        </div>
 
-        <div class="modal-footer">
 
-          <form class="form-inline" method="post" action="{% url "experiments-normandy-update" slug=experiment.slug %}">
+          <p>When the recipe is saved in Delivery Console, copy the Recipe ID below and hit save.</p>
```

Comment:
I like this layout, but I will say that errors really should go as close to the field they're talking about as possible.  I think you could put them either just above or just below their respective fields?

Reply:
ok I fixed the layout.